runtime/gc_modron_startup/mmparseXgc.cpp

```diff
@@ -1447,6 +1447,28 @@ gcParseXgcArguments(J9JavaVM *vm, char *optArg)
 			continue;
 		}
 
+		/* Check if there is a request a suballocator reservation increment size. */
+		if (try_scan(&scan_start, "suballocatorIncrementSize=")) {
+			if(!scan_udata_memory_size_helper(vm, &scan_start, &extensions->suballocatorIncrementSize, "suballocatorIncrementSize=")) {
+				returnValue = JNI_EINVAL;
+				break;
+			}
+			if(0 == extensions->suballocatorIncrementSize) {
+				j9nls_printf(PORTLIB, J9NLS_ERROR, J9NLS_GC_OPTIONS_VALUE_MUST_BE_ABOVE, "-Xgc:suballocatorIncrementSize=", (UDATA)0);
+				returnValue = JNI_EINVAL;
+				break;
+			}
+			continue;
+		}
+
+#if defined(LINUX)
```

Comment:
This condition should be exactly the same as in OMR MM_GCExtensionsBase" `if defined(OMR_ENV_DATA64) && defined(LINUX)`. Alternatively (and this is my preference) this field mighty exist for all platforms (despite is used on Linux only), at least in GC part of the code.

Reply:
Sounds good. I will just remove the directives from this patch and the omr patch.